bedrock/exp/templates/exp/firefox/new/download.html

```diff
@@ -2,50 +2,500 @@
+        <p>{{ ftl('firefox-desktop-download-firefox-automatically', attrs=automatically_attrs) }}</p>
+      </div>
+
+      <div class="mzp-c-card js-animate">
+        {{ picture('pinp', '143', 138) }}
+        <h3 class="mzp-u-title-xs">{{ ftl('firefox-desktop-download-picture-in-picture') }}</h3>
+        <p>{{ ftl('firefox-desktop-download-from-watching-a') }}</p>
+      </div>
+    </div>
+
+    {{ download_firefox(dom_id='download-discover', locale_in_transition=True, download_location='discover cta') }}
+  </section>
+
+  <section class="c-support">
+    {% set questions_attrs = 'href="https://support.mozilla.org/products/firefox%s&utm_content=mozilla-support" rel="external noopener" data-cta-type="link" data-cta-text="Mozilla support"'|safe|format(referrals) %}
+    {{ ftl('firefox-desktop-download-questions', attrs=questions_attrs) }}
+  </section>
+
+  <aside id="mobile-banner" class="show-android show-ios" data-nosnippet>
+    <div class="c-mobile">
+      <div class="c-mobile-text">
+        <h1 class="c-logo t-browser-word-hor-white-xs">{{ ftl('firefox-desktop-download-firefox-browser') }}</h1>
+        <h2 class="mzp-has-zap-7 mzp-u-title-md show-android">{{ ftl('firefox-desktop-download-get-firefox-android') }}</h2>
+        <h2 class="mzp-has-zap-7 mzp-u-title-md show-ios">{{ ftl('firefox-desktop-download-get-firefox-ios') }}</h2>
+
+        <p>{{ ftl('firefox-desktop-download-download-the-mobile') }}</p>
+
+        <div class="show-android">
+          {{ google_play_button() }}
+        </div>
+        <div class="show-ios">
+          <a href="{{ ios_url }}" data-link-type="download" data-download-os="iOS">
+            <img src="{{ l10n_img('firefox/ios/btn-app-store.svg') }}" alt="{{ ftl('download-button-download-app-store') }}" width="152" height="45">
+          </a>
+        </div>
+      </div>
+    </div>
+    <h2 class="c-desktop">
+      <a href="#next">{{ ftl('firefox-desktop-download-learn-about-the') }}</a>
+    </h2>
+    <span id="next"></span>
+  </aside>
+
+  {% if show_firefox_join_modal %}
+  <aside class="mzp-u-modal-content join-firefox-content">
+    <h4 class="join-firefox-title">{{ ftl('firefox-desktop-download-youve-already-got-the-browser') }}</h4>
+    <p class="join-firefox-intro">{{ ftl('firefox-desktop-download-watch-for-hackers-with') }}</p>
+
+    <a href="{{ url('firefox.accounts') }}" class="mzp-c-button mzp-t-product join-firefox-button" data-link-name="Get More From Firefox" data-link-type="button">{{ ftl('firefox-desktop-download-get-more-from-firefox') }}</a>
+
+    {{ download_firefox(dom_id='download-join-firefox-modal', alt_copy=ftl('firefox-desktop-download-just-download-the-browser'), button_color='mzp-t-secondary', locale_in_transition=True, download_location='other') }}
+  </aside>
+  {% endif %}
+
 </main>
 {% endblock %}
+
+{% block js %}
+  {{ js_bundle('firefox_desktop_download') }}
+
+  {% if show_firefox_join_modal %}
```

Comment:
This variable is not set anywhere, so the modal script isn't loading?

Reply:
I noticed this on the original page and did not want to remove it because I don't have the context for what it's for. Maybe @stephaniehobson knows more?

Reply:
The original page has the variable defined here: https://github.com/mozilla/bedrock/blob/master/bedrock/firefox/templates/firefox/new/desktop/download.html#L8

Reply:
Ah, okay. I will add that back in right now. I must've not realized what I was looking at.